src/rust/lib/frp/src/macros.rs

```diff
@@ -86,7 +86,7 @@ macro_rules! new_bridge_network {
 #[macro_export]
 macro_rules! _extend {
     ($trace:ident $network:ident $($ts:tt)*) => {
-        $crate::divide_on_terminator! { [[$crate::extend_lines] [$trace $network]] $($ts)* }
+        $crate::divide_on_terminator2! { [[$crate::extend_lines] [$trace $network]] $($ts)* }
```

Comment:
Too many 2 in macros names as for my taste. It makes the code more obscure that usually macros does.